nexus/types/src/inventory/display.rs

```diff
@@ -780,6 +788,27 @@ fn display_sleds(
     Ok(())
 }
 
+fn display_ntp_status(
+    ledgered_sled_config: &OmicronSledConfig,
+    ntp_timesync: &IdOrdMap<TimeSync>,
+    f: &mut dyn fmt::Write,
+) -> fmt::Result {
+    let timesync = ledgered_sled_config
+        .zones
+        .keys()
+        .find_map(|zone_id| ntp_timesync.get(zone_id));
+
+    match timesync {
+        None => writeln!(f, "No information from NTP for this sled")?,
```

Comment:
could I convince you to start this with lowercase to match the other lines of output?

Reply:
Done